src/Package/Impl/Packages/BasePackage.cs

```diff
@@ -58,6 +61,14 @@ public abstract class BasePackage<TLanguageService> : VisualStudio.Shell.Package
             foreach (var commmand in CreateMenuCommands()) {
                 menuCommandService.AddCommand(commmand);
             }
+
+            var dte = VsAppShell.Current.GetGlobalService<DTE2>(typeof(DTE));
+            var cbs = dte?.CommandBars;
+            var cb = dte?.CommandBars["R Toolbar"] as CommandBar;
```

Comment:
in VSCT only so it is not directly importable. VSCT can handle C++ .h files, but that's not for C#.